examples/hpa/README.md

```diff
@@ -1,3 +1,169 @@
+# custom-metrics-stackdriver-adapter example instructions
```

Comment:
At the top, add a link to the HPA docs: 

For further instructions, see https://cloud.google.com/stackdriver/docs/managed-prometheus/hpa

Reply:
done